cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -28,35 +28,15 @@ endif()
 
 #---Check for Zlib ------------------------------------------------------------------
 if(NOT builtin_zlib)
-  message(STATUS "Looking for ZLib")
   find_package(ZLIB)
   if(NOT ZLIB_FOUND)
-    message(STATUS "Zlib not found. Switching on builtin_zlib option")
```

Comment:
Isn't that a relevant message, esp if `-DZLIB_INCLUDE_DIR=...` etc was passed? Should it be kept?

Reply:
I added is back locally. I didn't mean to remove it. I was just considering to insert this into the new `find_package` macro, so that we don't have to add by hand everywhere.